Gemfile

```diff
@@ -4,9 +4,14 @@ source 'https://rubygems.org'
 
 gemspec
 
-gem 'rake', '~> 10.1'
+group :development, :test do
+  gem 'mutant',       '~> 0.11', '>= 0.11.22'
+  gem 'mutant-rspec', '~> 0.11', '>= 0.11.22'
+  gem 'rake',         '~> 13.0', '>= 13.0.6'
```

Comment:
__TODO:__ Extract rake upgrade *and* moving it into a group into (2) pre-commits.